tests/functional/final-harvest.test.js

```diff
@@ -130,20 +135,15 @@ testDriver.test('final harvest sends page action', workingSendBeacon, function (
 
   Promise.all([rumPromise, loadPromise])
     .then(() => {
-      t.equal(router.seenRequests.ins, 0, 'no ins harvest yet')
-
       let insPromise = router.expectIns()
 
       let loadPromise = browser
         .safeEval('newrelic.addPageAction("hello", { a: 1 })')
         .get(router.assetURL('/'))
 
-      return Promise.all([insPromise, loadPromise]).then(([ins, load]) => {
-        return ins
-      })
-    })
-    .then(() => {
-      t.equal(router.seenRequests.ins, 1, 'received one ins harvest')
+      return Promise.all([insPromise, loadPromise])
+    }).then((results) => {
+      t.ok(results[0].request.body, 'received ins harvest')
```

Comment:
I think all the counts are still relevant in final-harvest test; helps to detect unexpected payloads & behavior changes.

Reply:
Are you including the # of hits on the endpoints though?

Reply:
No the test handle is agnostic to the server and endpoint. If we feel like we really need to track that information for the beacon endpoints, we can.

Reply:
For example, the "final harvest sends multiple" test below used to track the resource and errors harvested at then-checkpoints. It seems to consider the matter of timing, and also if there were other errors aside from the one `noticeError` generated, it would've failed.